.devcontainer/Dockerfile

```diff
@@ -3,11 +3,22 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
 #-------------------------------------------------------------------------------------------------------------
 
-FROM golang:1.12.5
+FROM golang:1.12.5-stretch
```

Comment:
What's the difference between 1.12.5 and 1.12.5-stretch?
Why do you recommend 1.12.5-stretch? What functionally was missing?

Reply:
The `stretch` tag pins to a version of debian so it minimizes the chances of breaking changes when new debian releases come out. 

This is from the dockerhub page for the golang images:

> Some of these tags may have names like buster or stretch in them. These are the suite code names for releases of Debian and indicate which release the image is based on. If your image needs to install any additional packages beyond what comes with the image, you'll likely want to specify one of these explicitly to minimize breakage when there are new releases of Debian.

Reply:
Sounds good, happy to keep `-stretch` tag